html/welcome.html

```diff
@@ -0,0 +1 @@
+<p>Welcome to our App!</p>
```

Comment:
@bontgoy this mostly looks good but lets change the language some.

Try `Welcome to Phase 0 Tracks!`